SAMtags.tex

```diff
@@ -453,6 +455,112 @@ \subsubsection{Color space}
+this modification being correct, rather than the base being unmodified.
+
+\begin{description}
+\item[MM:Z:\tagregex{([ACGTN][-+][a-z](,[0-9]+)*;)*}]
+\hfill\\
+The first character is the unmodified base as seen in the {\sf SEQ}
+field, one of {\tt A}, {\tt C}, {\tt G}, {\tt T} or {\tt N}, with
+the exception that {\tt N} is used to match any base rather than
+strictly {\tt N}.  This is followed by either plus or minus
+indicating the strand the modification was observed on (relative to
+the original sequenced strand of {\sf SEQ} with plus meaning same
+orientation\footnote{Hence a tool that may reverse complement sequences does not need to understand how to manipulate the {\tt MM} and {\tt MP} tags.}), and a base modification symbol.
+This is then followed by a comma separated list of how many unmodified seq bases of the
+stated base type to skip, stored as a delta to the last and starting
+with 0 as the first (or next) base.  Hence this number series is
+comparable to the numbers in an {\tt MD} tag.
+
+For example {\tt C+m,5,12,0;} tells us there are three 5-Methylcytosine
+bases in the original {\sf SEQ}.  The first 5 {\tt C} bases are
+unmodified and the 6th is modified, as are the 19th (12 between the
+6th and 19th) and 20th.  Similarly {\tt G-m,14;} indicates the 15th
+{\tt G} is a 5-Methylcytosine on the opposite strand.
+
+This permits modifications to be listed on either strand with the rare
+potential for both strands to have a modification at the same site.
+
+Note it is permitted for the coordinate list to be empty (for example {\tt MM:Z:C+m;}), which may be used as an explicit indicator that this base modification is not present.
+It is not permitted for coordinates to be beyond the length of the sequence.
+
+If the modification is not one of the standard common types (listed
+below) it can be specified as a numeric ChEBI code.  For example
+{\tt C+76792,57;} is the same as {\tt C+h,57;}.
+
+An unmodified base of {\tt N} means count any base in {\sf SEQ}, not
+only those of {\tt N}.  Thus {\tt N+n,100;} means the 101st base is
+Xanthosine (n), irrespective of the sequence composition.
+
+The standard code types and their associated ChEBI values are listed
+below, taken from \emph{Modeling methyl-sensitive transcription factor
+motifs with an expanded epigenetic alphabet}, Coby Viner
+et.al. \url{https://www.biorxiv.org/content/10.1101/043794v1}.
+Additionally ambiguity codes {\tt A}, {\tt C}, {\tt G} and {\tt T}
+exist to represent unspecified modifications bases of their respective
+canonical base types, plus code {\tt N} to represent an unspecified
+modification of any base type.
+
+\begin{center}
+\begin{tabular}{lllll}
+{\bf Unmodified base} & {\bf Code} & {\bf Abbreviation} & {\bf Name} & {\bf ChEBI} \\
+\hline
+C & m & 5mC   & 5-Methylcytosine        & 27551 \\
+C & h & 5hmC  & 5-Hydroxymethylcytosine & 76792 \\
+C & f & 5fC   & 5-Formylcytosine        & 76794 \\
+C & c & 5caC  & 5-Carboxylcytosine      & 76793 \\
```

Comment:
chebi has this as carboxycytosine (no 'l')

Reply:
Impressive dedication to proof reading! :-)

I got these terms from the Viner presentation.  Googling shows both are in common usage, but Carboxylcytosine is considerably more common.  I'm unsure if that means the CHEBI database is in error or just using a different accepted spelling.

I'm out of my depth on this one basically.

Reply:
digging deeper,  it seems that this is the same thing: https://www.ebi.ac.uk/chebi/searchId.do?chebiId=46883